lib/bio/bio.c

```diff
@@ -51,6 +51,11 @@ static ssize_t bio_default_read(struct bdev *dev, void *_buf, off_t offset, size
     int err = 0;
     STACKBUF_DMA_ALIGN(temp, dev->block_size); // temporary buffer for partial block transfers
```

Comment:
While I'm in here, does this need to be aligned to a `block_size` or can we get away with `CACHE_LINE` ?